linera-service/src/project.rs

```diff
@@ -276,14 +328,23 @@ impl Project {
         Self::write_string_to_file(&state_path, STATE)
     }
 
-    fn create_contract_file(source_directory: &Path) -> Result<()> {
+    fn create_lib_file(source_directory: &Path) -> Result<()> {
+        let state_path = source_directory.join("lib.rs");
+        Self::write_string_to_file(&state_path, LIB)
+    }
+
+    fn create_contract_file(source_directory: &Path, project_name: &str) -> Result<()> {
+        let project_name = project_name.replace('-', "_");
         let contract_path = source_directory.join("contract.rs");
-        Self::write_string_to_file(&contract_path, CONTRACT)
+        let contract_contents = CONTRACT.replace("{project-name}", &project_name);
```

Comment:
If we're going to use regexp-search instead of `format!` strings, we should use a proper template framework in Rust. This way we may also have proper files on disk instead of quoting strings.